frameworks/cassandra/src/main/java/com/mesosphere/sdk/cassandra/scheduler/CassandraRecoveryPlanOverrider.java

```diff
@@ -113,7 +113,48 @@ private Phase getNodeRecoveryPhase(Plan inputPlan, int index) {
                 new UnconstrainedLaunchConstrainer(),
                 stateStore);
 
+        // Restart all other nodes if replacing a seed node to refresh IP resolution
+        List<Step> restartSteps = new ArrayList<>();
+        int replaceIndex = replaceStep.getPodInstanceRequirement().get().getPodInstance().getIndex();
+        if (isSeedNode(replaceIndex)) {
+            logger.info(
+                    "node-{} is a seed node. Restarting all other nodes to refresh seed node address.",
```

Comment:
Maybe go with `Scheduling restart of all other...`, I could see the current wording being cause for alarm